CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing to Intro to CPP

```

Comment:
What is CPP?

Reply:
Relic from source, updated to Isambard Docs